conf/messages

```diff
@@ -2121,8 +2121,8 @@ urr.bannertext = Get involved in making government services better (opens in a n
 urr.nothanks = No thanks
 
 ############################### LoginEvents ###############################
-login-event.title = You must provide new information for each responsible person
-login-event.heading = You must provide new information for each responsible person
+login-event.title = New information needed for each responsible person
+login-event.heading = New information needed for each responsible person
```

Comment:
@matthewharris1 was there a ticket for this?

Reply:
There wasn't a ticket for this, it was discussed in the sprint review demo we had yesterday afternoon. We decided this small content change was what we would go through with instead of creating 2 login events pages due to that taking up to much workload just for a user to see the page once.

Reply:
no ticket = no changes :P

Reply:
Ticket made, happy now?

Reply:
🥇